README.md

```diff
@@ -69,18 +81,19 @@ npm i
 
 To run this application locally, the first step is to create a local .env file and set the environment variables with the appropriate values. You can duplicate .env.example file and rename it to .env
 
+_Please note, in order run this project with your local database, you need an existing postgres username, and password. Some machines will have a superuser by default, when installed, but there may be cases where you need to create a role for postgres_.
+
 ```bash
 
 # src/common/.env.example
 
-PORT=3000
 BASE_URL=http://localhost:3000
 
-DATABASE_HOST=<Your Database host>
-DATABASE_NAME=<Database name, choose a name for your local database environment, this will be created automatically, when running the db:setup command>
-DATABASE_USER=<Your Database username, choose a name for the database role. A role will be created automatically based on what you add here>
-DATABASE_PASSWORD=<Your Database Password, choose a password, this is optional when creating a role for your local database>
-DATABASE_PORT=<Your Database Port>
+DATABASE_HOST=<Your Database hostname>
+DATABASE_NAME=<Database name - This will be the name of the database you will work with in this environment>
+DATABASE_USER=<Your Database username - Must be created in Postgres already, this can be a default role, or a role you have created>
+DATABASE_PASSWORD=<Must be created in Postgres already, if the Postgres role you selected has a password, you must add it here.>
+DATABASE_PORT=<Your Database Port - Default Port for postgres is 5432>
 
 
 Created for local testing, and CI environments. Locally, you can add the same values as above. But these values will be different when running tests in the github actions workflow.
```

Comment:
hmmm so the database we're using when running our e2e tests, and the one we are using when running the app is the same? Do you see any problems with that?

Reply:
Yes. I think one issue is that there may be some collisions with the database. If one database is being used for test and development then it can cause problems with data, there should be two different databases for testing and development.